docker-compose.yml

```diff
@@ -80,6 +80,7 @@ services:
 #    volumes:
 #      - .:/modyn_host
 #    shm_size: 4gb
+# CUDASTART (ignore/delete if enabling cuda manually)
```

Comment:
Just from an understanding point, this signs for cuda to include the following code?

Reply:
This is for the `initial_setup.sh` script :D To find the lines we are searching for reliably.